chanzuckerberg/idseq-web
# [Phylo Trees] Validate name on same step it is added

### Description

Validate the name of the phylo tree before moving on to the next step of phylo tree creation.

![Jun-10-2019 09-52-18](https://user-images.githubusercontent.com/37312125/59211763-87558f80-8b65-11e9-808b-a061d18561c0.gif)

### Testing

1. Check name of existing phylo trees
1. Click new phylo tree button
1. Choose any project and taxon
1. Input an existing phylo tree name
1. Click continue and verify invalid name error
1. Change phylo tree name and verify you can continue phylo tree creation.
